CfStreamUploader/CfStreamUploader.Presentation/ViewModel.cs

```diff
@@ -1,8 +1,135 @@
 ﻿using GalaSoft.MvvmLight;
+using GalaSoft.MvvmLight.Command;
 
 namespace CfStreamUploader.Presentation
 {
     public class ViewModel : ViewModelBase
     {
+        #region ColorChange
+
+        private bool isDarkmode = false;
+        public string themeText = "Lightmode";
+
+        private string baseColor = "Transparent";
+        private string contrastColor = "Transparent";
+        private string textColor = "Black";
+        private string borderBrush = "#6497e8";
+        private string button1Bg = "#6497e8";
+        private string button1Fg = "White";
+        private string button2Bg = "#6497e8";
+        private string button2Fg = "#6497e8";
+        private string button2FgMouseOver = "White";
+        private string progressColor = "Green";
+
+        public string ThemeText
```

Comment:
Write the private strings over the fields

Reply:
*fields before properties

Reply:
The private strings are fields. They are already at the top.